### DISCUSSION TITLE
Key bindings for navigating between opening file and terminal

### DISCUSSION THREAD

**Discussion by congdv** [zed-industries/zed#6740] [Feature Requests]
Hi there, I'm wondering if we have any key biding for these moves:
1. Navigating between current opening file and terminal
2. Navigating cross all opening files

Thank you for your help

**Comment by mrnugget** [comment]
> Navigating between current opening file and terminal

Does <kbd>Cmd-`</kbd> work for you? It's the "terminal panel: toggle focus" action you can open in command palette.

**Reply by op-roo** [reply]
I want to close the terminal by the same command ctrl-` . how can i do tat?

**Reply by mrnugget** [reply]
That works for me. I just hit <kbd>ctrl-`</kbd> again.

**Comment by congdv** [comment]
@mrnugget Yes it works. Thanks.

Also, we can use `Cmd-<number>` to navigate to other opening files tab
